bombastic/model/src/data.rs

```diff
@@ -80,14 +81,13 @@ impl SBOM {
                         match result.passed() {
                             true => return Ok(SBOM::CycloneDX(bom)),
                             false => {
-                                let all_reasons = result
-                                    .errors()
+                                let all_reasons = Self::get_validation_error_messages(result.clone())
```

Comment:
I think it just checks serial_number in here.[1] I think we can be able to directly use result.error('serial_number')."

[1] https://github.com/trustification/trustification/blob/b6b362c22c8bfbfce4235200c7a31e8723adc944/bombastic/model/src/data.rs#L101